python/src/dynamixel_sdk/protocol1_packet_handler.py

```diff
@@ -202,8 +210,9 @@ def rxPacket(self, port, rxpacket=[]):
         return rxpacket, result
 
     # NOT for BulkRead
-    def txRxPacket(self, port, txpacket, rxpacket=[], error=0):
```

Comment:
If you pass error=0 and then change it inside the function, then you modify the variable that is referenced by the label, not the same variable, since integers are immutable

Reply:
@roncapat 

oh, but as long as I know, the function used to return right error values. I should check this again

Reply:
@roncapat 

Kay